requirements.txt

```diff
@@ -1,4 +1,5 @@
 psycopg2-binary>=2.7,<3.0.0
+hdmf<=2.4.0
```

Comment:
As discussed in Slack, hdmf 2.5.1 reverts the major breaking change so we can do:

`hdmf!=2.5.0,<3.0.0`

Reply:
Already done :)